radixdlt/babylon-wallet-android
# [ABW-3435] - Display home cards carousel

## Description
This PR adds home cards carousel functionality.

## How to test
The users from organic app installs use-case:
1. Create a new wallet
2. The 'Connector' card will appear on the home screen at the top
